apps/greenhouse-management/src/components/Auth.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Spinner, Stack } from "juno-ui-components"
+import { useIsLoggedIn } from "./StoreProvider"
+
+// Adds a loading screen while during auth
+// Shows children when auth is complete
+
+const Auth = ({ children }) => {
+  const authLoggedIn = useIsLoggedIn()
+
+  return (
+    <>
+      {!!authLoggedIn && children}
+      {!authLoggedIn && (
+        <Stack
```

Comment:
We have already a component that takes care of the loding stuff. Whould you please use it?
https://github.com/sapcc/juno/blob/main/apps/greenhouse-management/src/components/shared/HintLoading.js